app/views/users/two_factor_setup/authenticator_setup/new.html.haml

```diff
@@ -0,0 +1,40 @@
+%article.verify
+  %h1 Google Authenticator setup
+
+  %h2 Step 1
+  %p
+    Enter this code into your authenticator app
+
+  .callout
+    %h3= @totp_secret
+
+  %section.sign-in-form
+    .sign-in-footer-links
+      .background-border-heading
+        %span.inner
+          Or
+
+  .qrcode
```

Comment:
Maybe - hard to say without some user research though. The code is the actual secret-key, so in this context it's the actual value being used to generate the TOTPs. 

But perhaps this is beside the point. I suppose the use-case is some folks _might_ write it down or store the key somewhere securely. Happy to shift it though if we think the QR code should be more prominent.